### PR TITLE
Remove Google docs, Google slides, Overleaf custom code

### LINKED ISSUE #42: Remove custom Google Docs code
There is some odd code in:

* https://github.com/jonathanpeppers/inclusive-code-comments/blob/main/src/common/googleDocs.js
* https://github.com/jonathanpeppers/inclusive-code-comments/blob/main/src/common/googleSlides.js

It doesn't seem like we want any custom code for Google Docs. Let's just delete it?

We'll probably need to test it doesn't break anything -- and maybe check what happens in Google Docs. We probably don't even need the extension to show up at all in Google Docs, but it's OK if it happens to work.

### PR DESCRIPTION
**Pull request by rachelkang**
Fixes #42

### CODE REVIEW

File: src/common/tweaksManager.js [hunk 1]
```diff
@@ -1,2 +1,2223 @@
...
...e=>a.getClassName(e)}}},{match:{hostname:"hey.com"},getInputAreaWrapperTweaks(e){const t=TweaksManager.DEFAULT_TWEAKS.getInputAreaWrapperTweaks(e);return{createMutationObserver:(e,a)=>t.createMutationObserver(e,a),getParsingDetector(e){const a=/^\s*On\s+(\w+)\s+\d+,\s+\d+,\s+.+\s+wrote:\s*$/,r=t.getParsingDetector(e);return{isIgnored:e=>r.isIgnored(e),isSignature:e=>r.isSignature(e),isQuote:e=>r.isQuote(e),getReplacementText:(e,t,a)=>r.getReplacementText(e,t,a),isBlock:(e,t)=>r.isBlock(e,t),createMutationObserver:e=>r.createMutationObserver(e),replaceText:(e,t,n)=>(e=e.replace(a,e=>"\ufeff".repeat(e.length)),r.replaceText(e,t,n)),isWhiteSpace:e=>r.isWhiteSpace(e)}},getFullTextInfo:(e,a)=>t.getFullTextInfo(e,a),getSelectedText:()=>t.getSelectedText(),getSelection:()=>t.getSelection()}}},{match:{url:/thunderbird.net|thunderbird-mail.de|\/thunderbird/},init(){const e=StorageController.create();e.onReady(()=>{e.updateStatistics({isThunderbirdUser:!0})}),TweaksManager.DEFAULT_TWEAKS.init()}},{match:{hostname:["clubdesk.com","clubdesk.de"]},supported:()=>!1,unsupportedMessage:()=>i18nManager.getMessage("siteCannotBeSupported")}],"undefined"!=typeof module&&(module.exports=TweaksManager);
```
Comment (rachelkang) [inline src/common/tweaksManager.js]: I reformatted this so I would be able to more easily navigate what to delete
Should I revert it back to this single line format ? Does the added spacing / lines affect the size of the project?

Comment (jonathanpeppers) [reply src/common/tweaksManager.js]: You did the right thing. For some reason, some of the original source code for this extension was minified.

When we run `npx xt-build` it minifies it for us.

**Comment by jonathanpeppers** [comment]
I should hopefully be able to test this tomorrow sometime, and we can merge. Looks good so far!

File: src/common/tweaksManager.js (hunk 1 above)
Comment (rachelkang) [reply src/common/tweaksManager.js]: good to know, thanks!

**Comment by rachelkang** [comment]
thanks @jonathanpeppers! it took me a while to figure out how to delete the right combination of things so that it still behaves as expected, but I can confirm now that it works well on my end!

**Review by rolfbjarne — APPROVED** [review]

**Review by jonathanpeppers — APPROVED** [review]
Seems to work, and I'm not seeing any JavaScript errors about something missing in the console. Thanks!

**Pull request merged**